apps/server/src/graphql/Task.ts

```diff
@@ -375,6 +392,52 @@ When the task is:
   })
 );
 
+builder.mutationField("makeTaskSubtask", (t) =>
+  t.prismaFieldWithInput({
+    type: "Task",
+    description: "Make a task a subtask of another task. Takes in the ID of the parent task and the ID of the subtask",
+    input: {
+      parentTaskId: t.input.id({ required: true, description: "The ID of the parent Task" }),
+      subTaskId: t.input.id({ required: true, description: "The ID of the subtask" }),
+    },
+    resolve: async (query, _, args) => {
+      const { parentTaskId, subTaskId } = args.input;
```

Comment:
nitpick: I prefer not to destructure inputs/params/props of a function/component and instead do `props.myProp` or `args.parentTaskId` in this case. I find it more readable to read `args.parentTaskId` than `parentTaskId`.